doc/source/cookbook/modulepaths-persist-over-sudo.rst

```diff
@@ -25,6 +25,7 @@ can be parsed to enable on the fly the default modulepaths (with ``module
 use`` Tcl modulefile command).
 
 .. literalinclude:: ../../example/modulepaths-persist-over-sudo/rc
+   :language: tcl
```

Comment:
For some reason, it shows an error for me when trying to render this with syntax highlighting, but specifying "tcl" here is necessary because it can deduce the language from the files' extension /because it has none).